tickets-test/src/main/java/no/unit/nva/publication/ticket/test/TicketTestUtils.java

```diff
@@ -36,7 +37,7 @@ public static Publication createPersistedPublication(PublicationStatus status, R
         var persistedPublication = Resource.fromPublication(publication).persistNew(resourceService,
                                                                                     UserInstance.fromPublication(
                                                                                         publication));
-        if (PublicationStatus.PUBLISHED.equals(status)) {
+        if (PublicationStatus.PUBLISHED.equals(status) || PublicationStatus.PUBLISHED_METADATA.equals(status)) {
```

Comment:
Probably use the same old Set::contains trick to keep it consistent

Reply:
fikset